index.yaml

```diff
@@ -15,23 +15,10 @@ indexes:
   - name: status
   - name: next_scheduled_check_time
 
-- kind: CollectionRightsSnapshotMetadataModel
```

Comment:
This file has many changes. Are they intended?

Reply:
Reverted.